pkg/blackduck/ctl_blackduck.go

```diff
@@ -383,3 +395,19 @@ func (ctl *Ctl) SpecIsValid() (bool, error) {
 func (ctl *Ctl) CanUpdate() (bool, error) {
 	return true, nil
 }
+
+// addEnvironToSpec adds an environ to the Spec's environs. It updates the environ to the new
+// value if it already exists
+func addEnvironToSpec(environ string, environList *[]string) []string {
```

Comment:
Could this be replaced with https://github.com/blackducksoftware/synopsys-operator/blob/master/pkg/util/helpers.go#L73 ?

Reply:
Nice catch! Done